src/StreamClient.cs

```diff
@@ -168,14 +168,20 @@ private static string GetRegion(StreamApiLocation location)
             {
                 case StreamApiLocation.USEast:
                     return "us-east";
-                case StreamApiLocation.Tokyo:
-                    return "tokyo";
                 case StreamApiLocation.Dublin:
                     return "dublin";
+                case StreamApiLocation.Tokyo:
+                    return "tokyo";
+                case StreamApiLocation.Mumbai:
+                    return "mumbai";
                 case StreamApiLocation.Singapore:
                     return "singapore";
                 case StreamApiLocation.Sidney:
                     return "sydney";
+                case StreamApiLocation.Oregon:
+                    return "oregon";
+                case StreamApiLocation.Ohio:
+                    return "Ohio";

```

Comment:
nitpick, but maybe lowercase here to be consistent? :smile:

Reply:
Missed that one.